openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -341,6 +345,20 @@ def test_get_account_unknown_user(self, api_client, user):
         response = client.get(reverse("accounts_api", kwargs={'username': "does_not_exist"}))
         assert 404 == response.status_code
 
+    @ddt.data(
+        ("client", "user"),
+    )
+    @ddt.unpack
+    def test_regsitration_activation_key(self, api_client, user):
+        self.create_regsitration_object(self.user)
+
```

Comment:
Add docstring on this unit test to explain why activation_key is a special case.